Simplenote/NoteListViewController.swift

```diff
@@ -55,6 +55,23 @@ class NoteListViewController: NSViewController {
     ///
     weak var searchDelegate: NoteListSearchDelegate?
 
+    var isActive: Bool = false {
+        didSet {
+            if isActive && searchField.currentEditor() != nil {
+                isActive = false
+            }
+
+            guard oldValue != isActive else {
+                return
+            }
+
+            for row in 0..<tableView.numberOfRows {
+                let rowView = tableView.rowView(atRow: row, makeIfNecessary: false) as? TableRowView
+                rowView?.isActive = isActive
+            }
```

Comment:
Nipticky: Maybe move to a method? (for consistency / cleanup purposes?)

Reply:
Done 👍